packages/contractkit/src/utils/azure-key-vault-client.ts

```diff
@@ -75,7 +70,10 @@ export class AzureKeyVaultClient {
     }
     const cryptographyClient = await this.getCryptographyClient(keyName)
     // @ts-ignore-next-line (ECDSA256 is not included in the client enum but is valid)
-    const signResult = await cryptographyClient.sign(this.SIGNING_ALGORITHM, message)
+    const signResult = await cryptographyClient.sign(
+      this.SIGNING_ALGORITHM,
+      new Uint8Array(message)
```

Comment:
This line was changed to fix the `value must be of type Uint8Array` issue. I believe some change in monorepo to our dependencies caused this to start failing.